src/Http/Controllers/VoyagerAuthController.php

```diff
@@ -34,7 +34,7 @@ public function postLogin(Request $request)
 
         $credentials = $this->credentials($request);
 
-        if ($this->guard()->attempt($credentials, $request->has('remember'))) {
+        if (app('VoyagerAuth')->attempt($credentials, $request->has('remember'))) {
```

Comment:
`VoyagerAuth` is just `auth()` ([by default](https://github.com/the-control-group/voyager/blob/1.2/src/VoyagerServiceProvider.php#L71)), so shouldn't it be `app('VoyagerAuth')->guard()`...?

If so, I'm not sure how this change fixes anything since it still uses the default guard.

Reply:
In my case, my project default is set to `api` and VoyagerAuth is now configured to use `web` instead.

```
$this->app->singleton('VoyagerAuth', function () {
    return Auth::guard('your-custom-guard');
});
```


Guard was changed like it says in the docs: [customization/custom-guard](https://docs.laravelvoyager.com/customization/custom-guard). Also all other default guard calls were already changed to use VoyagerAuth in #3968.

Reply:
On the other hand I'm also getting somewhat confused. As I submitted the PR I knew that this wasn't a fix for all my problems, but it allowed me to at least log in and it still seems to make much sense regarding changes made in #3968. 

But #3968 was made so that we can use a custom guard like it's also described in the docs. However since 1.2.0 Voyager is using Laravel's own Policies and Gate's which use the default guard anyway (`AuthorizesRequests`). If these assumptions are correct then it might need a lot more changes to make `VoyagerAuth` work like it's supposed to or it might be wise to remove `VoyagerAuth` completely from this project. Does any of this make sense?